sdk/ml/azure-ai-ml/tests/pipeline_job/e2etests/test_pipeline_job.py

```diff
@@ -69,6 +69,7 @@ def create_or_record_weekly_fixed_job_name(job_name: str):
 @pytest.mark.e2etest
 @pytest.mark.pipeline_test
 class TestPipelineJob(AzureRecordedTestCase):
+    @pytest.mark.skip("Skip for broken job service type, need fix")
```

Comment:
Better have task for this. As this breaks our public feature.

Reply:
Bug item 2076189